### ISSUE TITLE
Why replacing R17 as it works and module is calibrated for ?

### ISSUE THREAD

**Issue by Gjuju** [TheHWcave/Peacefair-PZEM-004T-#7]
As the title say, i'm confuse over the aim in changing R17 with R17a/R17b couple of resistors.

Is it just for a power dissipation reason ?

Or is there something i missed?

best regards.

**Comment by TheHWcave** [comment]
No, it is because of the voltage rating. The resistor has to sustain full mains voltage. The used resistor size is similar to resistors that are not rated for that kind  of voltage across them. It may arc over. By splitting the resistance value into  multiple resistors in series, each resistor has to withstand less voltage. For extra safety you may want to split into even more than just 2 resistors.  

    On Monday, 13 May 2024 at 20:33:42 BST, Julien Gainza ***@***.***> wrote:  
 
 


As the title say, i'm confuse over the aim in changing R17 with R17a/R17b couple of resistors.

Is it just for a power dissipation reason ?

Or is there something i missed?

best regards.

**Comment by ortegafernando** [comment]
@TheHWcave what about adding to existing R17 a simple 1 ohm (or even less) resistors in series after cutting the trace? will it works? (1Mohm vs 1 or 2 ohm is a extremely low value to affect calibration)

What do you think about just modifiynt the oposite? I am thinking about adding an ESP8266 (tasmota) and powering it from aC mains, (may be I need to change 7133-1 to a HT7333a (pin compatible and 30-40mA vs 250mA).

With this way, there will be not connection outside the module, so it will be safer for everyone

Do you know about this possibility? Thanks.
